src/MICore/Transports/StreamTransport.cs

```diff
@@ -147,6 +147,12 @@ protected void Echo(string cmd)
             lock (_locker)
             {
                 _writer?.WriteLine(cmd);
+                // https://github.com/Microsoft/MIEngine/issues/616 : If it is local gdb (MinGW/Cygwin) on Windows, we need to send an extra line after commands 
+                // so that if it errors, the error will come through. 
+                if((_callback is Debugger) && ((Debugger)_callback).IsLocalGdbOnWindows)
```

Comment:
@gregg-miskelly I can do that but the Logger will log all the newlines too making the logfile much longer.